snet_cli/arguments.py

```diff
@@ -325,6 +333,187 @@ def add_service_options(parser, config):
+        map(lambda x: x[len("network."):], filter(lambda x: x.startswith("network."), config.sections())))
+
+    org_list_p = subparsers.add_parser("list", help="List Organizations", default_choice="default")
+    org_list_p.set_defaults(fn="list")
+    networks_organization_subparsers = org_list_p.add_subparsers(title="networks", metavar="[NETWORK]")
+
+    for network_name in network_names:
+        p = networks_organization_subparsers.add_parser(network_name,
+                                                        help="List Organizations from {} network".format(network_name))
+        p.set_defaults(network_name=network_name)
+        _add_organization_arguments(p)
+
+    p = networks_organization_subparsers.add_parser("eth-rpc-endpoint",
+                                                    help="List Organizations at the provided Ethereum-RPC endpoint")
+    p.set_defaults(network_name="eth_rpc_endpoint")
+    p.add_argument("eth_rpc_endpoint", help="ethereum json-rpc endpoint (should start with 'http(s)://')",
+                   metavar="ETH_RPC_ENDPOINT")
+    _add_organization_arguments(p)
+
+    p = networks_organization_subparsers.add_parser("default")
+    _add_organization_arguments(p)
+
+    org_info_p = subparsers.add_parser("info", help="Organization's Informations", default_choice="default")
+    org_info_p.set_defaults(fn="info")
+    networks_organization_subparsers = org_info_p.add_subparsers(title="networks", metavar="[NETWORK]")
+    org_info_p.add_argument("name", help="Name of the Organization", metavar="ORG_NAME")
+
+    for network_name in network_names:
+        p = networks_organization_subparsers.add_parser(network_name,
+                                                        help="Organization's Informations from {} network".format(network_name))
+        p.set_defaults(network_name=network_name)
+        _add_organization_arguments(p)
+
+    p = networks_organization_subparsers.add_parser("eth-rpc-endpoint",
+                                                    help="Organization's Informations at the provided Ethereum-RPC endpoint")
+    p.set_defaults(network_name="eth_rpc_endpoint")
+    p.add_argument("eth_rpc_endpoint", help="ethereum json-rpc endpoint (should start with 'http(s)://')",
+                   metavar="ETH_RPC_ENDPOINT")
+    _add_organization_arguments(p)
+
+    p = networks_organization_subparsers.add_parser("default")
+    _add_organization_arguments(p)
+
+    org_create_p = subparsers.add_parser("create", help="Create an Organization", default_choice="default")
+    org_create_p.set_defaults(fn="create")
+    networks_organization_subparsers = org_create_p.add_subparsers(title="networks", metavar="[NETWORK]")
+    org_create_p.add_argument("name", help="Name of the Organization", metavar="ORG_NAME")
+    org_create_p.add_argument("members", help="List of members to be added to the organization",
+                              metavar="ORG_MEMBERS[]")
+
+    for network_name in network_names:
```

Comment:
As far as I understand for each organization command the list of networks is added as additional options for 
the command. On the other hand root parser already has --network option to specify the network. Can it be used here?

Reply:
Yes, I think we can get `network` from `config` (session).
I'll change it and make some tests to see if it's OK.